plugins/modules/database/influxdb/influxdb_retention_policy.py

```diff
@@ -208,15 +221,15 @@ def create_retention_policy(module, client):
         module.fail_json(msg="Failed to parse value of duration")
 
     influxdb_duration_format = parse_duration_literal(duration)
-    if influxdb_duration_format != 0 and influxdb_duration_format < 3600000000000:
+    if influxdb_duration_format != 0 and influxdb_duration_format < 1 * DURATION_UNIT_NANOSECS['h']:
```

Comment:
hi @xabinapal thanks for implementing those suggestions! :-) In this particular case, it *might* be slightly easier to read if we define a variable at the top of the function, like: `one_hour = 1 * DURATION_UNIT_NANOSECS['h']`, then the tests would become:
```suggestion
    if influxdb_duration_format != 0 and influxdb_duration_format < one_hour:
```

Reply:
I'm now thinking that it would make sense to define that constant at the top of the file, as is used both in `create_retention_policy` and `alter_retention_policy` functions.